app/controllers/state_file/questions/verification_code_controller.rb

```diff
@@ -55,8 +55,8 @@ def get_existing_intake(intake)
         state_intake_classes.each do |intake_class|
           key = intake.contact_preference == "text" ? :phone_number : :email_address
           search = intake_class.where.not(raw_direct_file_data: nil) # has a successful df import...
-          search.where(key => intake[key]) # ...using the same contact info
-          search.where.not(id: intake.id) if intake_class == intake.class # ...unless it's literally the same intake
+          search = search.where(key => intake[key]) # ...using the same contact info
+          search = search.where.not(id: intake.id) if intake_class == intake.class # ...unless it's literally the same intake
```

Comment:
may be worth a spec that the clause is picked up

Reply:
hmm yea, i mean i make one of the tests fail (pre-this code change) by adding the `first_existing_intake` in the spec file, but not sure if that's sufficient.

Reply:
@mrotondo helped me write more tests 🙏